.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "rust-analyzer.linkedProjects": [
+        "near-rs/contract/Cargo.toml",
+        "near-rs/integration-tests/Cargo.toml"
+    ]
+}
```

Comment:
I know this PR is for notes and not necessarily for merging, but if that changes please make sure to remove this

Reply:
I'd like to argue to the contrary, but it's a 1/4. lmk if you can be entertained by a new opinion.